### ISSUE TITLE
Unimplemented methods for validating Token Parameters

### ISSUE THREAD

**Issue by shahzaibj** [AzureAD/microsoft-authentication-library-for-android#677]
These methods don't seem to be implemented as they are not doing anything.

```
   protected void validateAcquireTokenParameters(AcquireTokenParameters parameters){
        return;
   }

   protected void validateAcquireTokenSilentParameters(AcquireTokenSilentParameters parameters){
        return;
   }
```

**File affected:** PublicClientApplication.java
**Branches affected:** dev, release/1.0.0

**Comment by iambmelt** [comment]
Partially resolved by #730

**Comment by shahzaibj** [comment]
This should now be completely resolved by #741 

The fix is already merged in release branch. I will update the issue once it is also in dev branch

**Comment by iambmelt** [comment]
Merged and done, thanks @shahzaibj
